src/components/Meme.js

```diff
@@ -13,7 +13,11 @@ export default function Meme() {
 	useEffect(function () {
 		fetch(api_url)
 			.then((data) => data.json())
-			.then((data) => setAllMemes(data.data.memes));
+			.then((data) => setAllMemes(data.data.memes))
+			// Handle the error when something wrong with the code
+			.catch((err) =>
+				document.write("<center> <h3>Engine can't understand this code, it's invalid. Please check code and reload </h3> </center> ")
+			);
```

Comment:
do not work on an issue until assigned. Remove this commit, then only I can merge you PR for original issue

Reply:
Accidentally pushed into this branch, sorry for the inconvenience. I have removed the commit.

Reply:
Please consider my latest commit.